### PR TITLE
Update kubeadm.md

### PR DESCRIPTION
**Pull request by Yannovitch**
Added code highlighting and sudo

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1497)

### CODE REVIEW

**Comment by Yannovitch** [comment]
I signed it!

**Comment by Yannovitch** [comment]
cla/linuxfoundation it is signed

**Comment by devin-donnelly** [comment]
Fine from a docs standpoint. Needs a tech review.

File: docs/getting-started-guides/kubeadm.md [hunk 1]
```diff
@@ -42,33 +42,37 @@ You will install the following packages on all the machines:
...
-      gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
-             https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
-      EOF
-      # setenforce 0
-      # yum install -y docker kubelet kubeadm kubectl kubernetes-cni
-      # systemctl enable docker && systemctl start docker
-      # systemctl enable kubelet && systemctl start kubelet
-
+      ```bash
+        sudo curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -`
```
Comment (errordeveloper) [inline docs/getting-started-guides/kubeadm.md]: You need unindent this also.

**Comment by errordeveloper** [comment]
@devin-donnelly LGTM, except for indentation.

**Comment by errordeveloper** [comment]
@Yannovitch, thank you very much for this, but it looks like more changes will be needed before we can merge it.

As @luxas pointed out  to me offline, in my last review I've not noticed that you are adding `sudo`. I am afraid it is not a good idea to use `sudo` like this. The instruction are specifically designed to run as root (i.e. `sudo -s` or `sudo su -`). Using `sudo` is not supported right now. The main problem with `sudo` is how it treats value of `HOME` environment variable, and resets other environment variables.

**Comment by jaredbhatti** [comment]
Marking this "tech review: open issues".

**Comment by jaredbhatti** [comment]
@Yannovitch  Any updates?

**Comment by jaredbhatti** [comment]
In the interest of PR cleanliness, I will close this issue in a week if there is no response.

**Comment by chenopis** [comment]
@Yannovitch This PR seems stale so I will close it. Please reopen it if you find another way to do it besides using sudo. Cheers

**Pull request closed without merging**